tap_github/__init__.py

```diff
@@ -414,7 +414,7 @@ def get_all_issue_milestones(schemas, repo_path, state, mdata):
                 # the GitHub API doesn't currently allow a ?since param for pulls
                 # once we find the first piece of old data we can return, thanks to
                 # the sorting
-                if bookmark_time and singer.utils.strptime_to_utc(r.get('due_on')) < bookmark_time:
+                if bookmark_time and r.get("due_on") and singer.utils.strptime_to_utc(r.get("due_on")) < bookmark_time:
```

Comment:
are we sure that records sorted by `due_on` field?
why can't we use something like `updated`?

Reply:
I only kept the previous logic. I am also not sure why `due_on` was chosen for this. Should I re-write it with `updated_at` field? There are other fields using `created_at` and `updated_at` which seems more consistent.

Reply:
agreed -- created/updated_at seems better here. Could you make this change? We'll need to make sure the cursor field output by the catalog is updated as well.